api_spec.yaml

```diff
@@ -1752,6 +1760,81 @@ paths:
                                 type: string
                                 enum:
                                     - Unauthorised access
+    /dashboard/api/tenants/third-party/boxy-saml:
```

Comment:
Think of a better url? it can be something like /dashboard/api/tenants/thirdparty/boxy-saml/clients-using-saml

Reply:
Changed it to: `dashboard/api/tenants/third-party/boxy-saml/clients-with-saml-metadata`